frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -106,9 +107,9 @@ export const funnelLogic = kea<funnelLogicType<TimeStepOption>>({
 
     loaders: ({ props, values, actions }) => ({
         results: [
-            [] as FunnelStep[],
+            [],
```

Comment:
Why remove this?

Reply:
Thanks! looks like i fat fingered something here. I'll add it back